src/poprox_recommender/topics.py

```diff
@@ -6,6 +6,9 @@
 
 from poprox_concepts import Article, ClickHistory
 from poprox_concepts.domain.topics import GENERAL_TOPICS
+from poprox_recommender.paths import model_file_path
+
+NEWS_MODEL = "news-category-classifier-distilbert"
```

Comment:
Though the code for topic matching exists in the repo, I don't think we're using it at all right now. Baking this model into the container image is an extra 400MB we don't currently need AFAIK

Reply:
This worked.